composer.json

```diff
@@ -56,14 +56,13 @@
         "jcalderonzumba/gastonjs": "^1.2",
         "jcalderonzumba/mink-phantomjs-driver": "^0.3.3",
         "metadrop/behat-contexts": "dev-fb-nuvoleweb-dev-i45-behat-4.x",
-        "metadrop/grumphp-drupal-check": "~0.1",
+        "metadrop/grumphp-drupal-check": "^0.3.0",
         "metadrop/scripthor": "1.x-dev",
-        "mglaman/drupal-check": "^1.1",
+        "mglaman/drupal-check": "^1.1.8",
```

Comment:
IMO it would be better to keep mglaman/drupal-check and phpro/grumphp dependencies on metadrop/grumphp package for better maintainability.

Reply:
Ok for mglaman/drupal-check, but I think we need grumphp even we don't use metadrop/grumphp-drupal-check. I will remove only mglaman/drupal-check